tests/test_date_pyparser.py

```
@@ -3,997 +3,1046 @@
+
+    # Compressed
+    for case in [('20000101', 2000,  1,  1),
+                 ('19991231', 1999, 12, 31),
+                 ('00000101',    0,  1,  1),
+                 (  '000101', 2000,  1,  1),
+                 (  '500630', 1950,  6, 30)]:
+        test, y, m, d = case
+        test = before + test + after
+        try:
+            pairs = parser_loose.parse_string(test).as_list()
+        except Exception as e:      # pragma: no cover
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)